### DISCUSSION TITLE
デザインの検討

### DISCUSSION THREAD

**Discussion by Mod12d** [Mod12d/relation-app#39] [General]
デザインやFigma関連のあれこれ

**Comment by yoko-at-home** [comment]
Design Day0
デザイン実装の前処理として、大枠の作成から着手したいと思います。
- アプリを開くとモーダルが開かれる
- モーダル内で、「トップ（ドット表示）/主な使用法/ HELP/説明」が横遷移できる
- User/Main 画面を作成する（ヘッダーのアイコンから遷移できる）

他にあればご指摘ください。（そもそもこの作業は#14かな🤔というのもあります）

**Comment by yoko-at-home** [comment]
実現できるかはさておき、こんなサイトを見つけました。
[Futuristic Web Design Inspirations](https://www.webdesign-inspiration.com/web-designs/style/futuristic)

**Comment by Mod12d** [comment]
ありがとうございます！
FIGMA側の修正もISSUEにしておきます（おいおいするという意味で） 🙏🏻

**Comment by yoko-at-home** [comment]
先日のミーティングで[こちらのサイ](https://photoshopvip.net/121220)トを紹介いただいたと思います。
[Lottie-reactライブラリ](https://www.npmjs.com/package/lottie-react)を使うと簡単にjsonからアニメーションを実装できそうなことがわかりました。
[Bがクルクル回るアニメーション](https://nextjs-bulletin-board-app.vercel.app/) 試してみました。使い所は分かりませんが...Loadingとか？🤔

**Comment by Mod12d** [comment]
そうです！
lottie使うイメージです

loading以外にも、効果として、さまざまなアクションに使用できます！
https://tech.smartcamp.co.jp/entry/2019/04/18/184533

**Comment by Mod12d** [comment]
できればこういったアイコンを使いたいなと
https://useanimations.com/

**Comment by yoko-at-home** [comment]
＠Mod12d さん、アイディアの宝庫✨
アニメーションアイコンかわいいですね〜！

[こんな雰囲気](https://www.youtube.com/watch?v=E5kctS3kLJE&list=PLLEsMtc8oEEyAS1wyzltOqNtkgYl53a3w&index=3)の近未来的な無機質な感じがお好みなのかなーと思っていたのですが、LottieアニメーションとかuseAnimationを使うとどちらかというと柔らかい雰囲気になりそうですね。

**Comment by Mod12d** [comment]
動画もかっこいいですね！
すみません、ちょっと僕も固まってないのですが、
技術的にはLottie使いたいかな〜とい気持ちがあります

**Comment by yoko-at-home** [comment]
動画というよりは、AEでモーショングラフィックを作成してjson書き出しするとLottieで使えるっぽい？と思いました。じゃあ作れるか？というと😅ですけれど。

**Comment by Mod12d** [comment]
アニメーションもOptionalなので、とりあえずPendingでも良いのかなと思います。
こちらタスクというdisuccusionなので、disucussionにconvertしますね🚀

**Comment by Mod12d** [comment]
メモ、Figmaのプロトタイプ機能
https://giginc.co.jp/blog/giglab/figma-animation
